### DISCUSSION TITLE
Generic relationships in prisma

### DISCUSSION THREAD

**Discussion by moaazsidat** [prisma/prisma#9997] [Q&A]
What is a good way to model generic relationships as they exist in SQLAlchemy (https://sqlalchemy-utils.readthedocs.io/en/latest/generic_relationship.html) in Prisma?

**Comment by austincrim** [comment]
Hey there @moaazsidat 👋 ,

We don't currently support this use case, but you might want to add your use case to [this issue](https://github.com/prisma/prisma/issues/5273) which is related to your question. That really helps us as we prioritize and research for our roadmap.

Thanks!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
